src/App.jsx

```diff
@@ -14,70 +16,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [product, setProduct] = useState('Jam');
+  const NO_GOODS_SELECTED = 'No goods';
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {product}
+        {product !== NO_GOODS_SELECTED ? ' is ' : ' '}
+        selected
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+        {product !== NO_GOODS_SELECTED && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setProduct(NO_GOODS_SELECTED)}
```

Comment:
you shouldn't set `'No goods'` string to `product` state because it's not a product. You may set an empty string (as a falsy string value - that indicates that nothing is stored now) as was proposed in the previous review, or `null` as a possible option.